apps/i18n/netsim/az_az.json

```diff
@@ -52,6 +52,8 @@
   "lobbyStatusWaitingForYou": "Waiting for you...",
   "logBrowserButton": "Log Browser",
   "logBrowserHeader": "All Router Logs",
+  "logBrowserHeader_all": "All Router Logs",
```

Comment:
Yup! [this doc](https://github.com/code-dot-org/code-dot-org/tree/staging/i18n#apps) gives the walkthrough